ecosystem/sep-0012.md

```diff
@@ -302,6 +340,41 @@ For example:
 }
 ```
 
+## Customer PUT Verification
+
+This endpoint allows servers to accept data values, usually confirmation codes, that verify a previously provided field via `PUT /customer`, such as `mobile_number` or `email_address`. Note that while fields such as `photo_proof_residence` or `notary_approval_of_photo_id` are verifications of other fields described in [SEP-9](#sep-0009.md), the server does not require the associated fields _before_ verification can be accomplished.
+
+### Request
+
+```
+PUT [KYC_SERVER || TRANSFER_SERVER]/customer/verification
+```
+
+Name | Type | Description
+-----|------|------------
+`id` | string | The ID of the customer as returned in the response of a previous `PUT` request.
```

Comment:
❗ Can you add a row here for the additional fields so all fields that can be sent are defined?
```suggestion
`id` | string | The ID of the customer as returned in the response of a previous `PUT` request.
`*_verification` | string | One or more fields that ...
```

Reply:
As specified in the request description:
> Data values used for verification should be included in the request using the SEP-9 attribute name plus `_verification`.

If you want me to add a single entry for `*_verification`, I can do that. But I don't think we should define a list of fields where verification is valid. We can leave that to the anchor to decide.

Reply:
I think we should add something, so adding something like `*_verification` at least makes it clear that's where the fields go.